examples/docker-in-docker-build-with-local-registry/.gitlab-ci-local-variables.yml

```
@@ -1,2 +1,2 @@
 ---
-CI_REGISTRY_IMAGE: "10.11.22.33:5000/docker-in-docker-build-with-local-registry"
+CI_REGISTRY_IMAGE: "gitlab-ci-local-registry:5000/docker-in-docker-build-with-local-registry"
```

to avoid hardcoding the image name and let CI_REGISTRY_IMAGE be defined out of CI_REGISTRY value.

```suggestion
CI_REGISTRY: "gitlab-ci-local-registry:5000"
```